certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -0,0 +1,173 @@
+# CD Renew JSON contract
+
+## Initial Request (PRELOADDATA):
+```json
+{
+  "rgState": "PRELOADDATA",
+  "powerOnFileName": "BANNO.CD.RENEW.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "0123456789S0123"},
+    {"id": 2, "value": ""},
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+ - userCharList[1]: 10-digit account number, 'S' and 4-digit share ID
+	 - Represents the target certificate or club share
+
+
+## Successful Response:
+```json
+{
+  "results": {
+    "currentState": {
+      "name": "My Wonderful CD",
+      "maturityDate": "01/01/21",
+      "minimumBalance": 12345.67,
+      "currentStatus": "Renew",
+      "transferSLId": "1234567890S0010",
+      "TransferAmount": 12345.67
+    },
+    "options": [{
+        "name": "Increase certificate balance",
+        "value": 1
+      },
+      {
+        "name": "Change certificate term",
+        "value": 2
+      },
+      {
+        "name": "Transfer certificate balance",
+        "value": 3
+      },
+      {
+        "name": "Renew",
+        "value": 4
+      },
+      {
+        "name": "Disburse funds by check",
+        "value": 5
+      },
+      {
+        "name": "Suspend",
+        "value": 6
+      }
+    ],
+    "transferShares": [{
+        "shareLoanId": "S0000",
+        "shareLoanName": "MY PRIME SHARE",
+        "shareBal": 1234.56
```

Comment:
Balances should be strings instead of numbers.

Reply:
Corrected in next commit